$OpenBSD: patch-shell_meson_build.orig,v 1.1 2021/10/30 09:14:56 ajacoutot Exp $

No GUdev on OpenBSD (XXX add HAVE_GUDEV to meson_options.txt)

Find libkrb5.so

Index: shell/meson.build
--- shell/meson.build.orig
+++ shell/meson.build
@@ -108,7 +108,6 @@ libshell_dep = declare_dependency(
 shell_sources = common_sources + files('main.c')
 
 shell_deps = common_deps + [
-  libdevice_dep,
   liblanguage_dep,
   libwidgets_dep,
   x11_dep,
@@ -130,7 +129,8 @@ executable(
          dependencies : shell_deps,
                c_args : cflags,
             link_with : panels_libs,
-              install : true
+              install : true,
+        install_rpath : '${LOCALBASE}/heimdal/lib'
 )
 
 
